src/components/Missions/Mission.jsx

```diff
@@ -0,0 +1,51 @@
+import Badge from "react-bootstrap/Badge";
+import PropTypes from "prop-types";
+import Button from "react-bootstrap/Button";
+import { useDispatch } from "react-redux";
+import { joinMission, leavingMission } from "../../redux/Missions/Missions";
```

Comment:
- [ ] Nice work done on your Mission component, kindly check this import `../../redux/Missions/Missions`, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux